script/os-autoinst-setup-multi-machine

```diff
@@ -157,7 +196,7 @@ main() {
     }
     ensure_ip_forwarding
     install_packages
-    configure_firewall
+    configure_nftables
```

Comment:
This leaves `configure_firewall` as unused function. Maybe there should be a switch to select which firewall to use?